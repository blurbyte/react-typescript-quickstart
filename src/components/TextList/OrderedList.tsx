import styled from 'styled-components';

import UnorderedList from './UnorderedList';

const List = UnorderedList.withComponent('ol');

const OrderedList = styled(List)`
  list-style-type: decimal;
`;

export default OrderedList;
