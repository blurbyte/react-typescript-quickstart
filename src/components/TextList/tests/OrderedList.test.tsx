import * as React from 'react';
import { shallow } from 'enzyme';

import OrderedList from '../OrderedList';

describe('OrderedList', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<OrderedList />);
    expect(wrapper).toMatchSnapshot();
  });
});
