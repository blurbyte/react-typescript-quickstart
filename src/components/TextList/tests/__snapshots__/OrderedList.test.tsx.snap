// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderedList renders correctly 1`] = `
.c0 {
  list-style-type: decimal;
}

<styled.ol
  className="c0"
/>
`;
